### ISSUE TITLE
Checklist for "big" release

### ISSUE THREAD

**Issue by grabbou** [reasonml-community/bs-react-native#251]
As discussed on the Discord channel, we are looking to release a big version next week.

Here's the checklist we have agreed upon:
- [x] Create a better project's documentation and "Getting Started" guide
- [x] Provide great changelog in a form of React article

Things that we should mention in the changelog:
- [x] All breaking changes and the upgrade path that happened after the last release
- [x] <s>The future of the project, including the "v2" that we have been prototyping</s>
- [x] Explicitly mention Discord group and invite everyone once again for collaboration!

It's mainly writing/documentation at this stage. There are a few more things to do to make the timing perfect.

**Comment by grabbou** [comment]
Cross-posting from another PR: 

- [ ] Fix status page and add proper links there
- [ ] Improve homepage with things we removed from Readme

**Comment by grabbou** [comment]
Work on status page is currently blocked by: https://github.com/ocaml/odoc/issues/200

Will submit a PR with the homepage, getting started and all other guides from Rebolt tomorrow.

Aiming for the release this Wednesday.

**Comment by grabbou** [comment]
Home page is finished, via #255

**Comment by czystyl** [comment]
The changelog discussion, via #256

**Comment by grabbou** [comment]
Decided we will talk about the future in a separate thread / article, once the initial version is released.

**Comment by grabbou** [comment]
Ok, this is now ready and published under `v0.10.0-rc.0`. Waiting for final review and making it public. Woo!
